hacspec/examples/aes-gcm/aes.rs

```diff
@@ -1,18 +1,29 @@
 // Import hacspec and all needed definitions.
 use hacspec::prelude::*;
+// TODO: move to hacspec_imports if we want to use it!
+use contracts::*;
```

Comment:
Well, this doesn't compile.
Can you drop the `contracts` for now?

Reply:
Sure, 
I will also add some tests